carbonserver/trie.go

```diff
@@ -1936,7 +1943,7 @@ func (ti *trieIndex) refreshUsage(throughputs *throughputQuotaManager) (files ui
 			cur.logicalSize += cur.node.meta.(*fileMeta).logicalSize
 			cur.physicalSize += cur.node.meta.(*fileMeta).physicalSize
 			cur.dataPoints += cur.node.meta.(*fileMeta).dataPoints
-
+			cur.readHits += cur.node.meta.(*fileMeta).readHits
```

Comment:
we also need to use atomic.LoadInt64 here too because unlike other metas, these is concurrent write.